content/en/docs/getting-started/examples.md

```diff
@@ -15,6 +15,26 @@ Use the [`openshift-e2e-aws`](https://steps.ci.openshift.org/workflow/openshift-
     workflow: openshift-e2e-aws
 {{< / highlight >}}
 
+# How do I write a simple "Execute this command in a container" test?
+
+Use a container test. Container tests are set up to always contain the source code, either by
+explicitly cloning it if they use an image that is in `base_images` or implicitly if they reference
+a `pipeline` image like `src`.
+
+{{< highlight yaml >}}
+base_images:
+  golangci-lint:
+    namespace: ci
+		name: golangci-lint
```

Comment:
nit: this mix of tabs and spaces causes weird rendering - let's use two spaces in lieu of tabs as needed?